modules/highgui/src/window.cpp

```diff
@@ -1082,6 +1082,16 @@ void cv::imshow(const String& winname, const ogl::Texture2D& _tex)
 #endif
 }
 
+std::string cv::currentUIFramework()
+{
+    auto backend = getCurrentUIBackend();
+    if (backend)
+    {
+        return backend->currentUIFramework();
+    }
+    return "";
```

Comment:
`return std::string();` instead of ""

---

Anyway where should be large `#if` switch for COCOA / QT and others "builtin" backends.
Check `cv::setWindowTitle` implementation as reference.